PowerEditor/src/WinControls/StatusBar/StatusBar.cpp

```diff
@@ -109,26 +108,28 @@ static LRESULT CALLBACK StatusBarSubclass(HWND hWnd, UINT uMsg, WPARAM wParam, L
 	{
 		case WM_ERASEBKGND:
 		{
-			// Skip background erasing and set fErase in PAINTSTRUCT instead,
-			// WM_PAINT does all the painting in all cases
-			return FALSE;
+			if (!NppDarkMode::isEnabled())
+			{
+				break;  // Let the control paint background the default way
+			}
+
+			RECT rc{};
+			::GetClientRect(hWnd, &rc);
+			::FillRect(reinterpret_cast<HDC>(wParam), &rc, NppDarkMode::getBackgroundBrush());
+			return TRUE;

```

Comment:
@yirkha 
It should return 0 according the doc:
https://learn.microsoft.com/en-us/windows/win32/winmsg/wm-erasebkgnd

In PR #15394, @ozone10 has explained as following:
https://github.com/notepad-plus-plus/notepad-plus-plus/pull/15394#issuecomment-2235347525 

So it's about a Windows procedure, its return value should follow the online document mentioned above.

Reply:
@donho 
`WM_ERASEBKGND` has specific returns.
>An application should return nonzero if it erases the background; otherwise, it should return zero.
It's okay here, we erase so it should be nonzero (`TRUE`).